test/Riok.Mapperly.IntegrationTests/UseUserMethodWithRefTest.cs

```diff
@@ -0,0 +1,30 @@
+using System.Threading.Tasks;
+using Riok.Mapperly.IntegrationTests.Helpers;
+using Riok.Mapperly.IntegrationTests.Mapper;
+using Riok.Mapperly.IntegrationTests.Models;
+using Shouldly;
+using VerifyXunit;
+using Xunit;
+
+namespace Riok.Mapperly.IntegrationTests
+{
+    public class UseUserMethodWithRefTest : BaseMapperTest
+    {
+        [Fact]
+        [VersionedSnapshot(Versions.NET6_0)]
```

Comment:
Does this snapshot really need to be versioned? The generated code is the same for all .net versions, isn't it?